modules/access_level/variables.tf

```diff
@@ -16,68 +16,75 @@
 
 variable "name" {
   description = "Description of the AccessLevel and its use. Does not affect behavior."
+  type        = string
 }
 
 variable "policy" {
   description = "Name of the parent policy"
+  type        = string
 }
 
 variable "combining_function" {
   description = "How the conditions list should be combined to determine if a request is granted this AccessLevel. If AND is used, each Condition must be satisfied for the AccessLevel to be applied. If OR is used, at least one Condition must be satisfied for the AccessLevel to be applied."
+  type        = string
   default     = "AND"
 }
 
 variable "description" {
   description = "Description of the access level"
-  default     = ""
+  type        = string
 }
 
 variable "ip_subnetworks" {
   description = "Condition - A list of CIDR block IP subnetwork specification. May be IPv4 or IPv6. Note that for a CIDR IP address block, the specified IP address portion must be properly truncated (i.e. all the host bits must be zero) or the input is considered malformed. For example, \"192.0.2.0/24\" is accepted but \"192.0.2.1/24\" is not. Similarly, for IPv6, \"2001:db8::/32\" is accepted whereas \"2001:db8::1/32\" is not. The originating IP of a request must be in one of the listed subnets in order for this Condition to be true. If empty, all IP addresses are allowed."
-  type        = "list"
-  default     = []
+  type        = list(string)
+  default     = null
```

Comment:
IMHO will be better to use as default [] for list

Reply:
Thank you, Ihor!
[Resolved](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/commit/63c5a8a98dbafd8436547a6d994293cafb99c825)